client/README.md

```diff
@@ -0,0 +1,3 @@
+# Running the frontend
+
+Run `npm start` in the client directory to run the frontend by itself.  Running `npm start` in the parent directory should also start the frontend server as well.  Requests that the client server cannot service will be automatically proxied to the backend.
```

Comment:
Can you clarify what "requests that the client server cannot service will be automatically proxied to the backend" means? I don't see anything that seems like it would do that.

Reply:
client package.json includes a "proxy" property that does this.